src/GuardClauses/GuardAgainstNullExtensions.cs

```diff
@@ -175,6 +198,30 @@ public static partial class GuardClauseExtensions
             return input;
         }
 
+#if NET5_0_OR_GREATER
+        /// <summary>
+        /// Throws an <see cref="ArgumentException" /> if <paramref name="input" /> is an empty or white space string.
+        /// </summary>
+        /// <param name="guardClause"></param>
+        /// <param name="input"></param>
+        /// <param name="parameterName"></param>
+        /// <param name="message">Optional. Custom error message</param>
+        /// <returns><paramref name="input" /> if the value is not an empty or whitespace string.</returns>
+        /// <exception cref="ArgumentException"></exception>
+        public static ReadOnlySpan<char> NullOrWhiteSpace(this IGuardClause guardClause,
+            ReadOnlySpan<char> input,
+            string parameterName,
+            string? message = null)
+        {
+            if (input.IsWhiteSpace())
```

Comment:
`MemoryExtensions.IsWhiteSpace(ReadOnlySpan<Char>)` Seems like a better fit?
https://learn.microsoft.com/en-us/dotnet/api/system.memoryextensions.iswhitespace?view=net-6.0

Reply:
It was changed (actually it is the same method but called via extension syntax, but explicit way looks preferred)